fabtests/functional/rdm_atomic.c

```diff
@@ -533,12 +534,16 @@ int main(int argc, char **argv)
 			}
 			break;
 		default:
+			if (!ft_parse_long_opts(op, argv[optind - 2],
```

Comment:
related to previous comment. no need to pass the opt in, lopt_idx should be enough to retrieve all the information. In addtion, 'optind - 2' assumes long opt has argument which is only true for this specific option.

Reply:
Thanks for your comment. 
Regarding this and your last comment, I was trying to do the option exact matching (disallowed the abbreviation) before. But think it more, it does not hurt to allow abbreviation as long as it's unique, so it can be simplified as `ft_parse_long_opts(int op, char *optarg)` without passing those two option name string.